pandas/types/concat.py

```diff
@@ -211,22 +211,23 @@ def convert_categorical(x):
         return Categorical(concatted, rawcats)
 
 
-def union_categoricals(to_union):
+def union_categoricals(to_union, sort_categories=False):
     """
     Combine list-like of Categoricals, unioning categories. All
-    must have the same dtype, and none can be ordered.
+    categories must have the same dtype.
 
     .. versionadded:: 0.19.0
 
     Parameters
     ----------
     to_union : list-like of Categoricals
+    sort_categories : boolean, default False
```

Comment:
I also prefer ``sort_categories`` to be explicit. 

BTW, I recommend @chris-b1 to open a separate PR on pandas master for open discussion.